### PR TITLE
i949: Allow API to connect to annex via jOOQ

### PR DESCRIPTION
**Pull request by MartinEden**
https://vimc.myjetbrains.com/youtrack/issue/VIMC-949

This branch doesn't compile, because it's missing the generated jOOQ code for the annex. The next PR is just the generated code, separated out for readability.

### CODE REVIEW

File: src/databaseTests/src/test/kotlin/org/vaccineimpact/api/databaseTests/tests/AnnexTests.kt [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package org.vaccineimpact.api.databaseTests.tests
```
Comment (littledice) [inline src/databaseTests/src/test/kotlin/org/vaccineimpact/api/databaseTests/tests/AnnexTests.kt]: Am I right in thinking if you remove this class for now (include in the second PR instead) the code will compile? Would be good not to have to merge in code with non passing tests!

**Comment by MartinEden** [comment]
Reshuffled as requested. :)

**Comment by littledice** [comment]
I'm confused! The code looks the same?

**Comment by MartinEden** [comment]
How about now?

**Comment by littledice** [comment]
Cool, will just wait for TC and then merge

**Pull request merged**